server/db/seeds/MainSeeder.ts

```diff
@@ -1,11 +1,21 @@
 import { DataSource } from "typeorm";
 import { runSeeder, Seeder, SeederFactoryManager } from "typeorm-extension";
-import { AdminRoleSeeder } from "./AdminRoleSeeder";
-import { UserRoleSeeder } from "./UserRoleSeeder";
+import { AdminRoleSeeder } from "./roles/AdminRoleSeeder";
+import { UserRoleSeeder } from "./roles/UserRoleSeeder";
+import { CanceledStatusSeeder } from "./statuses/CanceledStatusSeeder";
+import { CookingStatusSeeder } from "./statuses/CookingStatusSeeder";
+import { DeliveringStatusSeeder } from "./statuses/DeliveringStatusSeeder";
+import { ProcessingStatusSeeder } from "./statuses/ProcessingStatusSeeder";
+import { ReceivedStatusSeeder } from "./statuses/ReceivedStatusSeeder";
 
 export class MainSeeder implements Seeder {
     async run(dataSource: DataSource, factoryManager: SeederFactoryManager): Promise<void> {
         await runSeeder(dataSource, AdminRoleSeeder)
         await runSeeder(dataSource, UserRoleSeeder)
+        await runSeeder(dataSource, ProcessingStatusSeeder)
```

Comment:
I'm seeing a lot of duplication across seeders. My opinion is that we can use one seeder to seed all statuses. What do you think?